docs/pgbackup.md

```diff
@@ -1 +1,206 @@
 # pgbackup Project Specific Notes
+
+## Openstack Deployment
+If you want to deploy the pgbackup release under  an Openstack environment your need to follow the steps bellow.
+
+
+Be sure to first target your BOSH Director:
+```sh
+bosh target $BOSH_HOST
+```
+Now clone this release and cd into the directory:
+```sh
+git clone https://.../pgbackup-boshrelease.git
+cd pgbackup-boshrelease
+```
+If you intend on using a final release upload it like so:
+```sh
+bosh upload release releases/pgbackup-2.yml
+```
+
+Next edit the manifest file provided with the release (manifest/postgres-backup.yml).
+
+This release aims to support both v1 and v2 manifest. Currently, only Openstack templates are offered by default, 
+support for other Iaas are welcome.
+
+The first step is to choose under wish bosh you want to use de the postgres-backup release :
```

Comment:
consider the following ?

``You'll need to edit the manifest of the deployment to amend. You may choose to download the manifest from bosh director, or modify a version you saved into you version control (git?)``